server/routes/auth/auth.js

```diff
@@ -32,18 +40,23 @@ const getUserDetails = async (id, displayName) => {
 router.get('/auth/callback',
   passport.authenticate('OAuth2', { failureRedirect: '/bad' }),
   async function(req, res) {
-    // Successful authentication
+    // Successful 
     const user = await getUserDetails(req.user, req.user); // On whiten we'll take the display take from ping
 
-    // Check Ramad access
-    const isRamad = await checkIsAdmin(req.user);
-    req.session.admin = isRamad;
+    const { favorite_ads } = user;
+    const favoriteAds = favorite_ads[0] === null? [] : favorite_ads;
```

Comment:
`[0]`? I might miss something here 🤔

Reply:
The `array_agg` function return `[null]`, when the user dont have any favorite